### PR TITLE
[ MOSIP-35378 ] selected handles in uin generator stage. 

### PR DESCRIPTION
**Pull request by Khuddusshariff0022**

### CODE REVIEW

File: registration-processor/core-processor/registration-processor-uin-generator-stage/src/main/java/io/mosip/registration/processor/stages/uingenerator/stage/UinGeneratorStage.java [hunk 1]
```diff
@@ -87,6 +87,8 @@
 import io.mosip.registration.processor.status.dto.RegistrationStatusDto;
 import io.mosip.registration.processor.status.service.RegistrationStatusService;
 
+import javax.swing.plaf.IconUIResource;
```
Comment (ghost) [inline registration-processor/core-processor/registration-processor-uin-generator-stage/src/main/java/io/mosip/registration/processor/stages/uingenerator/stage/UinGeneratorStage.java]: Why is this import required?

Comment (Khuddusshariff0022) [reply registration-processor/core-processor/registration-processor-uin-generator-stage/src/main/java/io/mosip/registration/processor/stages/uingenerator/stage/UinGeneratorStage.java]: removed.

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]
